packages/core/test/assign.test.ts

```diff
@@ -261,94 +261,62 @@ describe('assign', () => {
   });
 });
 
+var value = globalThis.fetch;
+Object.defineProperty(globalThis, 'fetch', {
+  set(newValue) {
+    debugger;
+    value = newValue;
+  },
+  get() {
+    return value;
+  }
+});
```

Comment:
Hm?

Reply:
XDDDD haha, ye - let's pretend that this didn't happen :D